Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -597,6 +612,15 @@ public struct PackageBuilder {
             products.append(product)
         }
 
+        // Create a product for the entire package.
+        // FIXME: This is to be done only when parsing the manifest in Swift 3 mode.
+        if createImplicitProduct {
```

Comment:
That was explicitly rejected in the proposal, see: https://github.com/apple/swift-evolution/blob/master/proposals/0146-package-manager-product-definitions.md#inferring-implicit-products